test/views/commerce_slack_view_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule Aprb.Views.CommerceSlackViewTest do
 
   test "Transaction event renders transaction message" do
     event = Aprb.Fixtures.commerce_transaction_event()
-    slack_view = CommerceSlackView.render(event, "transaction.failed")
+    slack_view = CommerceSlackView.render(event, "transaction.failure")
```

Comment:
Is this where I should add tests for each fulfillment type (ship, pickup, foo) that can be passed to the transaction event? I was expecting to find a file in `./test/views/commerce/` and was surprised that there wasn't a file dedicated to the transaction view in there.

Reply:
oh good point! yes, it makes sense to divide these to their own files. I think this specific test is making sure if we call `CommerceSlackView.render` (the parent view) it calls the correct sub commerce render. I wonder what is the right way to test in this level, should we just check if `CommerceTransactionView.render` was called for this test here?